ec2/ec2test/availability_zones.go

```diff
@@ -17,15 +17,31 @@ import (
 	"gopkg.in/amz.v3/ec2"
 )
 
-// SetAvailabilityZones initializes the availability zones the test
+// SetAvailabilityZones replaces the availability zones the test
 // server is returning.
+//
+// NOTE: If zones does not contain one or more existing zones those
```

Comment:
This comment is confusing to me. "If zones does not contain... those zones are removed". If it doesn't contain them, how can they be removed?